node/heartbeat/export_test.go

```diff
@@ -70,3 +70,11 @@ func (hbmi *heartbeatMessageInfo) GetTotalDownTime() Duration {
 func (hbmi *heartbeatMessageInfo) GetIsActive() bool {
 	return hbmi.isActive
 }
+
+func Verify(hbmi *Heartbeat) error {
```

Comment:
Maybe rename this function to _VerifyLengths_ because _Verify_ is pretty general and can mean also verifying the signature or something like this.